spec/support/api/schemas/songs.json

```diff
@@ -29,22 +69,17 @@
           "duration": {
             "id": "http://spotify-apprentice-app.com/songs/0/duration",
             "type": "integer"
+          },
+          "artist_id": {
+            "id": "http://spotify-apprentice-app.com/songs/0/artist_id",
+            "type": "string"
           }
-        },
-        "required": [
-          "id",
-          "title",
-          "artist",
-          "album",
-          "duration"
-        ]
```

Comment:
You probably still want to require that id, title, album, and duration are present for a song.